eng/pipelines/test-deploy.yml

```diff
@@ -0,0 +1,32 @@
+resources:
+  repositories:
+  - repository: self
+    ref: $(branch)
+
+pr:
+  paths:
+    include:
+      - eng/pipelines/test-deploy.yml
+
+jobs:
+  - job: build_and_commit
+    displayName: Build and Commit
+    pool:
+      vmImage: 'ubuntu-latest'
+
+    steps:
+    - checkout: self
+
+    - task: UseNode@1
+      inputs:
+        version: '20.x'
+      displayName: 'Install Node.js'
+
+    - task: npmAuthenticate@0
+      inputs:
+        workingFile: ./website/.npmrc
```

Comment:
Should this step fail when something like this happen:

![image](https://github.com/Azure/awesome-azd/assets/24213737/fa0e1b8a-7c5d-468c-9d24-4e464cdf97e1)

https://dev.azure.com/azure-sdk/internal/_build/results?buildId=3486370&view=logs&j=b4175db5-e0ef-56c3-a737-ea868b7fdf52&t=fbcb7db1-d609-577b-a2de-3a6f80fe2225&l=10